diff -ruN netbsd/usr.bin/find/extern.h find/extern.h
--- netbsd/usr.bin/find/extern.h	2020-04-10 23:58:13.512335998 +0900
+++ find/extern.h	2020-04-12 21:10:18.390139676 +0900
@@ -31,7 +31,7 @@
  *	from: @(#)extern.h	8.3 (Berkeley) 4/16/94
  */
 
-#include <sys/cdefs.h>
+//#include <bsd.h>
 
 void	 brace_subst(char *, char **, char *, int *);
 PLAN	*find_create(char ***);
diff -ruN netbsd/usr.bin/find/find.c find/find.c
--- netbsd/usr.bin/find/find.c	2020-04-10 23:58:13.512335998 +0900
+++ find/find.c	2020-04-12 21:10:18.390139676 +0900
@@ -32,15 +32,6 @@
  * SUCH DAMAGE.
  */
 
-#include <sys/cdefs.h>
-#ifndef lint
-#if 0
-static char sccsid[] = "from: @(#)find.c	8.5 (Berkeley) 8/5/94";
-#else
-__RCSID("$NetBSD: find.c,v 1.30 2016/06/13 00:04:40 pgoyette Exp $");
-#endif
-#endif /* not lint */
-
 #include <sys/types.h>
 #include <sys/stat.h>
 
@@ -53,6 +44,7 @@
 #include <stdlib.h>
 #include <stdbool.h>
 #include <unistd.h>
+#include <bsd.h>
 
 #include "find.h"
 
diff -ruN netbsd/usr.bin/find/function.c find/function.c
--- netbsd/usr.bin/find/function.c	2020-04-10 23:58:13.513335998 +0900
+++ find/function.c	2020-04-12 22:34:27.699239815 +0900
@@ -32,17 +32,9 @@
  * SUCH DAMAGE.
  */
 
-#include <sys/cdefs.h>
-#ifndef lint
-#if 0
-static char sccsid[] = "from: @(#)function.c	8.10 (Berkeley) 5/4/95";
-#else
-__RCSID("$NetBSD: function.c,v 1.77 2018/09/04 15:16:15 kre Exp $");
-#endif
-#endif /* not lint */
-
 #include <sys/param.h>
 #include <sys/stat.h>
+#include <sys/statvfs.h>
 #include <sys/wait.h>
 #include <sys/mount.h>
 
@@ -61,7 +53,7 @@
 #include <string.h>
 #include <tzfile.h>
 #include <unistd.h>
-#include <util.h>
+#include <bsd.h>
 
 #include "find.h"
 
@@ -78,6 +70,16 @@
 	}								\
 }
 
+struct netbsd_statvfs {
+	struct statvfs buf;
+	unsigned long	f_flag;
+	char	f_fstypename[_VFS_NAMELEN];
+};
+
+#define	statvfs		netbsd_statvfs
+
+int	statvfs(const char *__restrict, struct statvfs *__restrict);
+
 static	int64_t	find_parsenum(PLAN *, const char *, const char *, char *);
 static	void	run_f_exec(PLAN *);
 	int	f_always_true(PLAN *, FTSENT *);
@@ -469,12 +471,14 @@
 		errx(1, "-delete: %s: relative path potentially not safe",
 			entry->fts_accpath);
 
+#if 0
 	/* Turn off user immutable bits if running as root */
 	if ((entry->fts_statp->st_flags & (UF_APPEND|UF_IMMUTABLE)) &&
 	    !(entry->fts_statp->st_flags & (SF_APPEND|SF_IMMUTABLE)) &&
 	    geteuid() == 0)
 		chflags(entry->fts_accpath,
 		       entry->fts_statp->st_flags &= ~(UF_APPEND|UF_IMMUTABLE));
+#endif
 
 	/* rmdir directories, unlink everything else */
 	if (S_ISDIR(entry->fts_statp->st_mode)) {
@@ -962,7 +966,7 @@
 {
 	u_int32_t flags;
 
-	flags = entry->fts_statp->st_flags;
+	flags = entry->fts_statp;
 	if (plan->flags == F_ATLEAST)
 		return ((plan->f_data | flags) == flags);
 	else
diff -ruN netbsd/usr.bin/find/ls.c find/ls.c
--- netbsd/usr.bin/find/ls.c	2020-04-10 23:58:13.513335998 +0900
+++ find/ls.c	2020-04-12 21:10:18.390139676 +0900
@@ -29,17 +29,9 @@
  * SUCH DAMAGE.
  */
 
-#include <sys/cdefs.h>
-#ifndef lint
-#if 0
-static char sccsid[] = "from: @(#)ls.c	8.1 (Berkeley) 6/6/93";
-#else
-__RCSID("$NetBSD: ls.c,v 1.21 2011/08/31 16:24:57 plunky Exp $");
-#endif
-#endif /* not lint */
-
 #include <sys/param.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 
 #include <err.h>
 #include <errno.h>
@@ -52,6 +44,7 @@
 #include <time.h>
 #include <tzfile.h>
 #include <unistd.h>
+#include <bsd.h>
 
 #include "find.h"
 
diff -ruN netbsd/usr.bin/find/main.c find/main.c
--- netbsd/usr.bin/find/main.c	2020-04-10 23:58:13.513335998 +0900
+++ find/main.c	2020-04-12 21:10:18.391139676 +0900
@@ -32,17 +32,6 @@
  * SUCH DAMAGE.
  */
 
-#include <sys/cdefs.h>
-#ifndef lint
-#if 0
-static char sccsid[] = "@(#)main.c	8.4 (Berkeley) 5/4/95";
-#else
-__COPYRIGHT("@(#) Copyright (c) 1990, 1993, 1994\
- The Regents of the University of California.  All rights reserved.");
-__RCSID("$NetBSD: main.c,v 1.31 2013/01/24 17:50:08 christos Exp $");
-#endif
-#endif /* not lint */
-
 #include <sys/types.h>
 #include <sys/stat.h>
 
@@ -56,6 +45,7 @@
 #include <string.h>
 #include <time.h>
 #include <unistd.h>
+#include <bsd.h>
 
 #include "find.h"
 
diff -ruN netbsd/usr.bin/find/misc.c find/misc.c
--- netbsd/usr.bin/find/misc.c	2020-04-10 23:58:13.513335998 +0900
+++ find/misc.c	2020-04-12 21:10:18.391139676 +0900
@@ -32,15 +32,6 @@
  * SUCH DAMAGE.
  */
 
-#include <sys/cdefs.h>
-#ifndef lint
-#if 0
-static char sccsid[] = "from: @(#)misc.c	8.2 (Berkeley) 4/1/94";
-#else
-__RCSID("$NetBSD: misc.c,v 1.14 2006/10/11 19:51:10 apb Exp $");
-#endif
-#endif /* not lint */
-
 #include <sys/types.h>
 #include <sys/stat.h>
 
diff -ruN netbsd/usr.bin/find/operator.c find/operator.c
--- netbsd/usr.bin/find/operator.c	2020-04-10 23:58:13.513335998 +0900
+++ find/operator.c	2020-04-12 21:10:18.391139676 +0900
@@ -32,15 +32,6 @@
  * SUCH DAMAGE.
  */
 
-#include <sys/cdefs.h>
-#ifndef lint
-#if 0
-static char sccsid[] = "from: @(#)operator.c	8.1 (Berkeley) 6/6/93";
-#else
-__RCSID("$NetBSD: operator.c,v 1.10 2014/10/18 08:33:30 snj Exp $");
-#endif
-#endif /* not lint */
-
 #include <sys/types.h>
 
 #include <err.h>
diff -ruN netbsd/usr.bin/find/option.c find/option.c
--- netbsd/usr.bin/find/option.c	2020-04-10 23:58:13.513335998 +0900
+++ find/option.c	2020-04-12 21:10:18.391139676 +0900
@@ -32,15 +32,6 @@
  * SUCH DAMAGE.
  */
 
-#include <sys/cdefs.h>
-#ifndef lint
-#if 0
-static char sccsid[] = "from: @(#)option.c	8.2 (Berkeley) 4/16/94";
-#else
-__RCSID("$NetBSD: option.c,v 1.27 2016/06/13 00:04:40 pgoyette Exp $");
-#endif
-#endif /* not lint */
-
 #include <sys/types.h>
 #include <sys/stat.h>
 
